libres/lib/enkf/enkf_main.cpp

```diff
@@ -556,8 +549,8 @@ static void enkf_main_add_ensemble_members(enkf_main_type *enkf_main) {
    is mainly to be able to test that the site config file is valid.
 */
 
-enkf_main_type *enkf_main_alloc(const res_config_type *res_config, bool strict,
-                                bool verbose, bool read_only) {
+enkf_main_type *enkf_main_alloc(const res_config_type *res_config, bool verbose,
```

Comment:
Could you write a sentence or two about what `strict` is and why it is no longer needed?

Reply:
strict is a parameter that was passed but never used/invoked. don't know if it's there to future proof the function or is a lingering parameter from it's past implementation. but eitherway it is not needed and might even confuse the use of the function. Since the documentation state a actual (non-existing) use of the parameter.